website/docs/patterns/multi-select-patterns/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -0,0 +1,52 @@
+When combining a Multi-select pattern with filtering, sorting, pagination, etc, you _will_ encounter confusing scenarios about how selected results should be persisted across different pages and what the user expects when traversing large data sets.
+
+These interaction guidelines are _not exhaustive_, but instead should serve as conversation starters between design and engineering for how best to handle the user experience.
+
+!!! Info
+
+If you run into unique scenarios that these guidelines don’t account for, we would love to hear about them through our normal [support](/about/support) channels.
+!!!
+
+## Consistent communication
+
+The most important method to ensure users are interacting with the results they intend to is to clearly communicate what they have selected, or even that a selection of results exists, **at all times**.
+
+This can help to prevent unnecessary deletions of results or making unintended changes if what is currently selected is unclear.
+
+## Persisting the selection
+
+One of the most common questions around Multi-select patterns stems from when and how selected results are persisted when a user performs different actions on a Table. Consider the following scenario:
+
+A user is viewing a data set consisting of 10,000 results that is presented in a Table. In order to make changes to specific results the user may take the following actions:
+
+1. **Filter** the Table to display only a subset of the results.
+2. **Sort** the Table by a specific parameter to order the results in a meaningful way.
+3. Traverse through multiple pages of the Table through **Pagination**.
+
+How the selection is persisted across all of these actions can be confusing and open the user up to making unintended changes if what they have selected is persisted (or not persisted) in a logical manner.
+
+### Within a paginated Table
+
+If a user makes a selection on one page of a paginated Table and progresses to the next page, persisting this selection can be communicated by the [selected count](/patterns/multi-select-patterns?tab=guidelines#selected-count) and can be helpful in scenarios where much of the data is similar or overlaps.
+
+<video loop controls width="100%">
+    <source
+        src="/assets/patterns/multi-select-patterns/multi-select-pagination-interaction.mp4"
+        type="video/mp4"
+    />
+</video>
+
+### When filtering Table results
```

Comment:
It may be good to communicate that these are conversation starters for product teams and that we don't (and probably won't) have answers for them to these questions.

Reply:
+1

Reply:
I do reference that at the top of this page, but I've included a reiteration of this point.